server/cron/calendar_cron.go

```diff
@@ -5,32 +5,24 @@ import (
 	"errors"
 	"fmt"
 	"slices"
-	"strconv"
-	"strings"
 	"sync"
 	"time"
 
-	"github.com/fleetdm/fleet/v4/ee/server/calendar"
 	"github.com/fleetdm/fleet/v4/server/config"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/fleetdm/fleet/v4/server/service/calendar"
 	"github.com/fleetdm/fleet/v4/server/service/schedule"
 	"github.com/go-kit/log"
 	kitlog "github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 )
 
 const (
-	calendarConsumers  = 18
-	defaultDescription = "needs to make sure your device meets the organization's requirements."
-	defaultResolution  = "During this maintenance window, you can expect updates to be applied automatically. Your device may be unavailable during this time."
+	calendarConsumers = 18
+	reloadFrequency   = 12 * time.Hour
```

Comment:
I kept it as a backup in case there is an issue with the new callback code. We could get rid of it, but since it is already working, it seemed ok to keep it.
Also we need reload for backward compatibility with current events.

Reply:
Yeah, good idea.